node/pkg/db/governor_test.go

```diff
@@ -92,13 +95,13 @@ func TestIsTransfer(t *testing.T) {
 }
 
 func TestIsPendingMsg(t *testing.T) {
-	assert.Equal(t, true, IsPendingMsg([]byte("GOV:PENDING:"+"2/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/789101112131415")))
+	assert.Equal(t, true, IsPendingMsg([]byte("GOV:PENDING2:"+"2/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/789101112131415")))
 	assert.Equal(t, false, IsPendingMsg([]byte("GOV:XFER:"+"2/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/789101112131415")))
-	assert.Equal(t, false, IsPendingMsg([]byte("GOV:PENDING:")))
-	assert.Equal(t, false, IsPendingMsg([]byte("GOV:PENDING:"+"1")))
-	assert.Equal(t, false, IsPendingMsg([]byte("GOV:PENDING:"+"1/1/1")))
-	assert.Equal(t, false, IsPendingMsg([]byte("GOV:PENDING:"+"1/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/")))
-	assert.Equal(t, true, IsPendingMsg([]byte("GOV:PENDING:"+"1/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/0")))
```

Comment:
Might be nice to have a negative case with a valid and invalid PENDING v1 example.

Reply:
And ensure they equal `false`

Reply:
Add a couple more cases.